### ISSUE TITLE
github-page

### ISSUE THREAD

**Issue by nttstar** [albumentations-team/albumentations.ai#16]
An excellent technical website theme!
Does `albumentations.ai` fully support GitHub-page? I just want to make sure about this as I have seen the `github-page` button on the right side.

**Comment by creafz** [comment]
Hey @nttstar 
Not sure if I understood your question correctly. Code in this repository builds a static site that is deployed to GitHub pages. So https://albumentations.ai is fully served from GitHub pages.

**Comment by nttstar** [comment]
@creafz Yes thank you! Then I realized that ``make prod`` can generate the final static HTML files for Github-page hosting.
